genesis-builder/framework/evm/sources/Evm.move

```diff
@@ -15,13 +16,17 @@ module Evm::evm {
 
     const OWNER: address = @evm_admin;
 
-    /// TODO: what capabilities should this have?
-    struct EvmLog has drop {
+    struct EvmLog has copy, store, drop {
         addr: address,
         topics: vector<u256>,
         data: vector<u8>,
     }
 
+    #[event]
+    struct EvmLogsEvent has store, drop {
```

Comment:
Why is it wrapped like that?

Reply:
Honestly I am not sure what it does, but I [saw it in the Aptos framework](https://github.com/aptos-labs/aptos-core/blob/aptos-node-v1.14.0/aptos-move/framework/aptos-framework/sources/fungible_asset.move#L185) as a way of marking structs you wanted to emit as events, so I did it too.